scripting/shavit-misc.sp

```diff
@@ -2081,7 +2081,7 @@ public Action Shotgun_Shot(const char[] te_name, const int[] Players, int numCli
 	TE_WriteVector("m_vecOrigin", temp);
 	TE_WriteFloat("m_vecAngles[0]", TE_ReadFloat("m_vecAngles[0]"));
 	TE_WriteFloat("m_vecAngles[1]", TE_ReadFloat("m_vecAngles[1]"));
-	TE_WriteNum("m_iWeaponID", TE_ReadNum("m_iWeaponID"));
+	TE_WriteNum("m_weapon", TE_ReadNum("m_weapon"));
```

Comment:
`m_weapon` is CS:GO only. `m_iWeaponID` still exists in CS:S. You might want to do this instead:

```sp
if(gEV_Type == Engine_CSS)
{
	TE_WriteNum("m_iWeaponID", TE_ReadNum("m_iWeaponID"));
}

else
{
	TE_WriteNum("m_weapon", TE_ReadNum("m_weapon"));
}
```
Also, have you verified that other than getting rid of errors, using `m_weapon` still serves the purpose of the code?